fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2167,4 +2167,6 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static long max_per_node_grep_log_limit = 500000;
 
+    @ConfField
+    public static boolean enable_execute_script_on_frontend = true;
```

Comment:
set default value false and mutable is true?

Reply:
the purpose of this config is to allow customers with strict security concerns to disable this function entirely. If it's mutable, a user who wants to use this can change its value to true, then execute script.